tools/accuracy_checker/accuracy_checker/metrics/character_recognition.py

```diff
@@ -37,3 +37,22 @@ def evaluate(self, annotations, predictions):
 
     def reset(self):
         self.accuracy.reset()
+
+
+class LabelLevelRecognitionAccuracy(PerImageEvaluationMetric):
+    __provider__ = 'label_level_recognition_accuracy'
+
+    annotation_types = (CharacterRecognitionAnnotation, )
+    prediction_types = (CharacterRecognitionPrediction, )
+
+    def configure(self):
+        self.accuracy = AverageEditdistanceMeter(lambda annotation, prediction: editdistance.eval(prediction, annotation))
```

Comment:
probably, if it is only one metric which use `AverageEditdistanceMeter` you can use default realization without lambda. Just make it the default loss